.circleci/config.yml

```diff
@@ -44,53 +44,73 @@ jobs:
           command: *store_stripe_version
 
       - restore_cache:
+          name: Restore Fabric SDK
           keys:
             - fabric-sdk-cache-{{ checksum "fabric_version.txt" }}
 
       - restore_cache:
+          name: Restore Stripe SDK
           keys:
             - stripe-sdk-cache-{{ checksum "stripe_version.txt" }}
 
       - run:
-          name: Store Xcode Version
+          name: Cache Xcode Version
```

Comment:
I think this should still be `Store Xcode Version` it does the same thing as `Store Stripe Version` and `Store Fabric SDK Version` above. It's basically writing the Xcode version to a file so that it can later be `checksum`'d as a part of the cache key for restoring the Carthage cache.

This just works around a shortcoming in CircleCI's caching mechanism in that you can't access custom environment vars in the `yml` file 😕